src/ProtocolInfo.php

```diff
@@ -32,11 +32,11 @@ private function __construct(){
 	 */
 
 	/** Actual Minecraft: PE protocol version */
-	public const CURRENT_PROTOCOL = 729;
+	public const CURRENT_PROTOCOL = 748;
 	/** Current Minecraft PE version reported by the server. This is usually the earliest currently supported version. */
-	public const MINECRAFT_VERSION = 'v1.21.30';
+	public const MINECRAFT_VERSION = 'v1.21.40';
 	/** Version number sent to clients in ping responses. */
-	public const MINECRAFT_VERSION_NETWORK = '1.21.30';
+	public const MINECRAFT_VERSION_NETWORK = '1.21.40';
```

Comment:
I know it's not normally very useful but change to 1.21.41

Reply:
That's not necessary because it's the same as 1.21.40 and 1.21.41